techmatters/flex-plugins
# Conferencing: put service user on hold when conferencing starts

## Description
This PR adds logic to put all service users in hold before an external party is called.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1961)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts

### Verification steps
- Start Flex development server.
- Start a voice contact and accept with your Flex instance.
- Add an external party to the call.
- Confirm that before the call to the external party starts, the service user (child) is put in hold.
